config.json

```diff
@@ -679,6 +679,14 @@
     },
     {
       "core": false,
+      "difficulty": 6,
+      "slug": "tournament",
+      "topics": null,
```

Comment:
If you want to add some topics to this exercise then that would be great :) It's fine if you don't want to though, I can add it to the list of exercises we still need to add topics to :)